custom_components/opnsense/switch.py

```
@@ -354,17 +355,25 @@ def _handle_coordinator_update(self) -> None:
+        if self._rule is None:
             return
-        await self._client.enable_filter_rule_by_created_time(self._tracker)
-        await self.coordinator.async_refresh()
+
+        async def enable_filter_rule():
+            await self._client.enable_filter_rule_by_created_time(self._tracker)
+            await self.coordinator.async_refresh()
+
+        await queue.put(enable_filter_rule)
 
     async def async_turn_off(self, **kwargs: Any) -> None:
```

Pylint (3.3.7)

[warning] Unused argument 'kwargs'

(W0613)